### DISCUSSION TITLE
Mount over itself

### DISCUSSION THREAD

**Discussion by Anuskuss** [rfjakob/gocryptfs#864] [General]
Is it safe to mount the decrypted directory over the encrypted one? I would like to only have a single folder but I'm not sure if it's safe. Will it (in the worst case) just confuse the driver or will it outright damage my data?

**Comment by kaoneko** [comment]
How will you even achieve that? gocryptfs outright refuses to mount the cipherdir on itself:

```
$ gocryptfs -nonempty cipherdir cipherdir
Mountpoint "/path/to/cipherdir" would shadow cipherdir "/path/to/cipherdir", this is not supported
```

**Reply by Anuskuss** [reply]
I'm sure you can make it happen somehow I'm just interested in the overall risk.

**Comment by rfjakob** [comment]
It's not supported as it would make it more complicated for gocryptfs to
access the encrypted files. See also:
https://github.com/rfjakob/gocryptfs/discussions/718
